docs/docs/forms.md

```diff
@@ -76,6 +76,8 @@ handleChange(event) {
 }
 ```
 
+However, if `this.state.value` becomes `null` or `undefined` at any time (including the initial state), React will evaluate the component as uncontrolled and may produce a warning: `Input elements should not switch from uncontrolled to controlled (or vice versa)`.
+
```

Comment:
Instead of including the specific copy of the warning, maybe it would be better to focus on explaining how to handle empty values. Something like:

> You should use an empty string to represent an empty controlled input. Setting a controlled input's value to `null`, or `undefined` will cause React to assume the component is no longer controlled and may produce a warning.